tests/baselines/reference/verbatimModuleSyntaxRestrictionsESM(esmoduleinterop=false).errors.txt

```diff
@@ -2,17 +2,6 @@
 /main.ts(3,8): error TS1259: Module '"/decl"' can only be default-imported using the 'allowSyntheticDefaultImports' flag
 
 
-==== /main.ts (2 errors) ====

```

Comment:
So weird that the file's output moved in the baseline. It moved down to the bottom in this one, but up to the top in `.types`.

Reply:
Yeah, I stared at that for a good 5 minutes trying to figure out if I had screwed something up.